src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => { // eslint-disable-line
```

Comment:
**[optional]**
Do not modify linter rules, by doing so you miss out on the goal, that particular rules seeks to teach you. Here you are trying to disable the javascript `eval()` which when disabled and misused can exposed your code to serious injection attacks. [2](https://masteringjs.io/tutorials/eslint/ignore), [2](https://eslint.org/docs/rules/no-eval)